src/angular/styles/_typography.scss

```diff
@@ -2762,6 +2764,7 @@ $panelBorderRadius: 2;
   position: relative;
   top: var(--sbb-box-shadow-horizontal-border-offset-top-below);
   box-shadow: var(--sbb-box-shadow-below), var(--sbb-box-shadow-horizontal-border-bottom);
+  border-color: var(--sbb-color-weak-accent);
```

Comment:
The border color is already set in line 2757. I suggest to...
* remove the line above and change line 2757 to `border: var(--sbb-border-width) solid var(--sbb-color-weak-accent);`
* remove `--sbb-panel-border-color`

Reply:
replacing --sbb-panel-border-color with --sbb-color-weak-accent would change the border color in the dark mode from smoke to milk, is that intended?

Reply:
Yes, I think that's what we discussed yesterday. And that's what your current code does.
Maybe I misunderstand something. We can also discuss this in a short call.